OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -66,8 +85,24 @@ SEARCH_APP_USER_PASSWORD=<Search User Password>
 Run the following command to create the Admin Server container:
 
 ```
-$ docker run -i -t --name $ADMIN_SERVER_CONTAINER_NAME --network=WCPortalNET -p <Any Free Port>:$ADMIN_PORT -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <directory>/webcenter.env.list $WCPortalImageName
+$ docker run -i -t --name <ADMIN_CONTAINER_NAME> --network=<NETWORK_NAME> -p <HostFreePort>:<ADMIN_PORT> -v <DATA_VOLUME>:/u01/oracle/user_projects --env-file <webcenter.env.list> oracle/wcportal:12.2.1.4
 ```
+Sample Run Command
```

Comment:
Please add a blank line before and after this line and only "Sample" needs a capital letter.